tests/manage/test_verify_rbd_based_rwo_pvc.py

```diff
@@ -0,0 +1,210 @@
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def test_fixture(request):
+    """
+    Setup and teardown
+    """
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def setup(self):
+    """
+    Create new pod using available PVC
+    """
+    # Create Storage Class with reclaimPolicy: Delete
+    self.sc_obj = helpers.create_storage_class(
+        interface_type=constants.CEPHBLOCKPOOL,
+        interface_name=self.cbp_obj.name,
+        secret_name=self.secret_obj.name,
+        reclaim_policy='Delete'
+    )
+
+    # Create PVC with 'accessModes' 'ReadWriteOnce'
+    pvc_data = templating.load_yaml_to_dict(constants.CSI_PVC_YAML)
+    pvc_data['metadata']['name'] = helpers.create_unique_resource_name(
+        'test', 'pvc'
+    )
+    pvc_data['metadata']['namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
+    pvc_data['spec']['storageClassName'] = self.sc_obj.name
+    pvc_data['spec']['accessModes'] = ['ReadWriteOnce']
+    self.pvc_obj = PVC(**pvc_data)
+    self.pvc_obj.create()
+
+    # Create two pods
+    log.info(f"Creating two pods which use PVC {self.pvc_obj.name}")
+    pod_data = templating.load_yaml_to_dict(constants.CSI_RBD_POD_YAML)
+    pod_data['metadata']['name'] = helpers.create_unique_resource_name(
+        'test', 'pod'
+    )
+    pod_data['metadata']['namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
+    pod_data['spec']['volumes'][0]['persistentVolumeClaim']['claimName'] = (
+        self.pvc_obj.name
+    )
+    self.pod_obj = helpers.create_pod(**pod_data)
+    pod_data['metadata']['name'] = helpers.create_unique_resource_name(
+        'test', 'pod'
+    )
+    self.pod_obj2 = helpers.create_pod(wait=False, **pod_data)
+
+
+def teardown(self):
+    """
+    Delete Storage Class
+    """
+    self.sc_obj.delete()
+
+
+@tier1
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
+    create_ceph_block_pool.__name__,
+    test_fixture.__name__
+)
+@pytest.mark.polarion_id("OCS-533")
+class TestRbdBasedRwoPvc(ManageTest):
+    """
+    Verifies RBD Based RWO Dynamic PVC creation
+    """
+    def test_rbd_based_rwo_pvc_reclaim_delete(self):
+        """
+        Verifies RBD Based RWO Dynamic PVC creation with Reclaim policy set to
+        Delete
+
+        Steps:
+        1. Create two pods using same PVC
+        2. Run IO on first pod
+        3. Verify second pod is not getting into Running state
+        4. Delete first pod
+        5. Verify second pod is in Running state
+        6. Verify usage of volume in second pod is matching with usage in
+           first pod
+        7. Run IO on second pod
+        8. Delete second pod
+        9. Delete PVC
+        10. Verify PV associated with deleted PVC is also deleted
+        """
+        # Run IO on first pod
+        log.info(f"Running IO on first pod {self.pod_obj.name}")
+        usage = run_io_and_verify_mount_point(
```

Comment:
from the pr #334 we can use run_io from pod object, this is better than dd, so lets wait for that to be merged

Reply:
PR is merged, if you can now make use of that method, check the example here please: https://github.com/red-hat-storage/ocs-ci/blob/master/tests/manage/test_pvc_deletion_during_io.py#L56-L69

Reply:
Done